test/fuzz/param.ml

```diff
@@ -14,3 +14,7 @@ let initial_fuel = 100
 let allow_partial_timeout = true
 
 let max_time_execution = 0.01 (* seconds *)
+
+let save_modules = true (* Set to false to disable saving modules *)
+
+let output_dir = "generated_modules" (* Directory to save modules *)
```

Comment:
`let output_dir = Fpath.v "generated_modules"` (this is better than using strings to represents paths)